CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# Release 2.18.0 16-09-2020
+
+- Add name prefix for backend/jwtProvider cluster (#330)
+- Split specifying root certs for backend vs sidestream SSL clients (#325)
+- Rich access logging of http request information (#316)
+- Support IP in backend address (#323)
+- For Cloud Run, report location with region instead of zone (#314)
```

Comment:
Please add this one too: https://github.com/GoogleCloudPlatform/esp-v2/pull/318

Reply:
Done.